packages/react/package.json

```diff
@@ -21,15 +28,18 @@
     "test:unit:watch": "jest --watch"
   },
   "dependencies": {
+    "@aws-amplify/ui-core": "^0.0.1",
+    "@aws-amplify/ui-theme-base": "^0.0.1",
```

Comment:
I'll move theme-base into core in another PR. We won't lose tree-shaking or anything, but it'll be 1 less bundle for customers to keep in sync.